dissect/target/plugins/apps/vpn/wireguard.py

```diff
@@ -102,37 +102,40 @@ def config(self) -> Iterator[Union[WireGuardInterfaceRecord, WireGuardPeerRecord
 
             config = _parse_config(config_buf)
 
-            for section in config.sections():
+            for section, config_dict in config._sections.items():
```

Comment:
Not sure relying on a "private" field of a Python standard library module is a great idea here. Is there no other way to achieve this? Perhaps a `functools.partial`?